.gitignore

```diff
@@ -33,4 +33,8 @@ bcipy/parameters/parameters_*
 bcipy_cache
 bcipy/language/lms/lm_dec19_char_large_12gram.*
 bcipy/language/lms/out_*
-bcipy/language/out/
\ No newline at end of file
+bcipy/language/out/
+
+
+bcipy/simulator/tests/resource/
+!bcipy/simulator/data
```

Comment:
`!`?

Reply:
We have `/data` earlier in the file, but this was too general and ignoring the data module in the simulator directory. The `!` indicates not, so don't ignore this one.